docs/BigQueryTable-batchsink.md

```diff
@@ -53,8 +53,8 @@ is ignored if the table already exists.
 * When this is set to true, table will be created with required partition filter. 
 * When this is set to false, table will be created without required partition filter.
 
-**Clustering order**: Clustering order determines the sort order of the data. Clustering organizes data 
-based on the contents of specified columns in the schema into optimally-size storage blocks, which 
+**Clustering order**: Determines the sort order of the data. Clustering organizes data 
```

Comment:
I cannot find the doc now. I have read that every description should start with a noun. It should answer the question "What is this property?". 
If you see any examples https://github.com/data-integrations/google-cloud/blob/develop/docs/GCS-batchsink.md this is usually how they do it.